okio/src/main/java/okio/Buffer.java

```diff
@@ -644,7 +644,7 @@ int selectPrefix(Options options) {
       Buffer data = new Buffer();
       copyTo(data, 0, Math.min(32, size));
       throw new EOFException("\\n not found: size=" + size()
-          + " content=" + data.readByteString().hex() + "...");
+          + " content=" + data.readByteString().hex() + "…");
```

Comment:
Guava recently moved away from non-ASCII in toString. I forget the reasons, will have to look it up.